tests/storage/exception_test.py

```diff
@@ -43,6 +43,10 @@ def find_module_exceptions(module, base_class=None):
         if base_class and not issubclass(obj, base_class):
             continue
 
+        # Internal classes (starting with '_' by convention) are skipped.
```

Comment:
"(starting ...)" is not needed, people know that names starting with _ are private.